public/app/routes/routes.ts

```diff
@@ -537,6 +537,24 @@ export function setupAngularRoutes($routeProvider: route.IRouteProvider, $locati
       controllerAs: 'ctrl',
       reloadOnSearch: false,
     })
+    .when('/graph/d/pmm-dbaas/cluster', {
```

Comment:
Why the route changes and new components if we are going to send to an existing dashboard URL? 🤔

Reply:
Valid, I removed the routes altogether!